README.md

```diff
@@ -17,29 +17,35 @@ You can install the provider using `pip`:
 pip install qiskit-alice-bob-provider
 ```
 
-`pip` will handle installing all the python dependencies automatically and you
-will always install the  latest (and well-tested) version.
+`pip` will handle installing all the python dependencies automatically and 
```

Comment:
"will install the latest release with all its dependencies automatically"

Reply:
Fixed